cli/compose/loader/loader_test.go

```diff
@@ -985,6 +985,84 @@ services:
 	assert.Error(t, err, `invalid mount config for type "bind": field Source must not be empty`)
 }
 
+func TestLoadBindMountSourceIsWindowsAbsolute(t *testing.T) {
+	tests := []struct {
+		doc      string
+		yaml     string
+		expected types.ServiceVolumeConfig
+	}{
+		{
+			doc: "C-drive lowercase",
+			yaml: `
+version: '3.3'
+
+services:
+  windows:
+    image: mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2019
+    volumes:
+      - type: bind
+        source: c:\
```

Comment:
Just to be safe, can the unit tests use `E:\` or `X:\` as the source path's drive (something other than `C:\`) in some of the cases?

Reply:
Yes, I can change that; let me update